### ISSUE TITLE
A11y_Visual Studio Code Services_Docs_Left Right Navigation_Keyboard : The dropdown icon is not visible for keyboard users in the left navigation section.

### ISSUE THREAD

**Issue by jadhavprathmesh01** [microsoft/vscode-docs#5101]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**

#A11yMAS; #A11yTCS; #A11ySev2; #DesktopWeb; #Visual Studio Code Services; #BM-Visual Studio Code Services-Web-Jan2022; #Web; #Keyboard;#WCAG2.1.1; #FTP; #ChromiumEdge;

**Environment Details:**
Application Name: Visual Studio Code Services
Windows Version: Windows 11

**Repro Steps:**

1. Launch the application Visual Studio Code Services
2. Tab navigate to "Docs" link.
3. Left and right navigation will appear.
4. Tab navigate to the Left navigation links and observe that the dropdown icon is visible or no using keyboard.

**Actual:**
There are dropdowns in the "Left navigation" section and the dropdown icon is not visible when the user navigates to the links using keyboard, when we hover using mouse 

**Expected Result:**
There should be dropdown icon visible when the keyboard user navigates to the links.

**User Impact:**
The keyboard only users will face difficulty if the dropdown icon is not visible.

**Attachment:**
![A11y_Visual Studio Code Services_Docs_Left Right Navigation_Keyboard](https://user-images.githubusercontent.com/87858881/149892665-cc176d75-d0ae-41f3-a65a-2775554ea161.gif)

**Comment by gregvanl** [comment]
We deliberately want to avoid too much visual clutter in the left navigation. It should be clear to readers that those items are section heading and the Enter key allows users to expand and collapse the sections.

**Comment by jadhavprathmesh01** [comment]
@gregvanl The dropdown icon are visible on mouse hover so basically it should be visible on keyboard focus also. Opening an new bug because this bug was closed. Please don't close the accessibility bugs as the issue is still reproducing.

**Comment by gregvanl** [comment]
@jadhavprathmesh01 How do we handle `won't fix` issues? Do we leave the issues open in our repo?

Also can you provide a link to documentation to explaining why we need to add a dropdown icon. We believe it will add visual clutter to the existing UI. Thanks

**Comment by jadhavprathmesh01** [comment]
@gregvanl As it is failing the 1.3.1 std i.e. the visual is not matching the programmatic information that is been given by screenreader and the screenreader is announcing it but for keyboard users its not available

And if you have any more questions you can join Indian office hours meeting.
